stock_picking_auto_create_lot_qty/tests/test_stock_picking_auto_create_lot_qty.py

```diff
@@ -58,12 +58,22 @@ def test_multiples_allowed_false(self):
         )
         self.assertEqual(len(lot), 2)
 
-    def test_multiples_allowed_true(self):
+    def test_multiples_allowed_true_1(self):
         self._create_picking()
         self._create_move(
             product=self.product_serial_auto_qty_5_multiples_allowed_true, qty=8.0
         )
         # check for error
+        # reminder = 3>0
+        self.assertRaises(UserError, self.picking.button_validate)
+
+    def test_multiples_allowed_true_2(self):
+        self._create_picking()
+        self._create_move(
+            product=self.product_serial_auto_qty_5_multiples_allowed_true, qty=15.0
+        )
+        # check for error
+        # reminder = 0
         self.assertRaises(UserError, self.picking.button_validate)
```

Comment:
question: Maybe I'm misunderstanding something, but

- multiples of 5 are allowed
- the quantity is 15
- this shouldn't raise any error

no?

Reply:
You're right 
Thanks, I'll fix it